Src/FluentAssertions/Collections/StringCollectionAssertions.cs

```diff
@@ -290,5 +290,49 @@ private bool NotContainsMatch(string wildcardPattern)
                 return !scope.Discard().Any();
             });
         }
+
+        /// <summary>
+        /// Expects the current collection of strings to have all elements in ascending order. Elements are compared
+        /// using the given <see cref="IComparer{T}" /> implementation.
```

Comment:
Should the `IComparer` part be left out or changed to refer to the provided lambda expression?